services/channel/workflows/package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "channel-workflows",
+  "version": "1.0.0",
+  "private": true,
+  "description": "",
+  "author": "Axinom",
+  "license": "MIT",
+  "keywords": [
+    "axinom",
+    "mosaic",
+    "axinom mosaic"
+  ],
+  "scripts": {
+    "clean": "rimraf dist .cache",
+    "build": "pilet build --source-maps=false",
+    "start": "pilet debug --port 10250",
+    "upgrade": "pilet upgrade --npm-client yarn",
+    "dev": "yarn start",
+    "dev:full": "yarn util:load-vars pilet debug --feed '\\$MICRO_FRONTEND_SERVICE_BASE_URL/v1/pilets/\\$TENANT_ID/\\$ENVIRONMENT_ID' --port 10250",
+    "codegen": "yarn util:load-vars graphql-codegen --config codegen.yml",
+    " -- utilities -- ": "",
+    "util:load-vars": "env-cmd -f .env env-cmd -f ../../../.env cross-env-shell"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "dependencies": {
+    "@apollo/client": "^3.7.10",
+    "@axinom/mosaic-managed-workflow-integration": "0.15.0-rc.8",
+    "@axinom/mosaic-ui": "0.47.0-rc.8",
+    "clsx": "^1.2.1",
+    "env-cmd": "^10.1.0",
+    "formik": "^2.2.9",
+    "graphql": "^15.4.0",
+    "graphql-tag": "^2.12.6",
+    "i18n-iso-countries": "^6.8.0",
+    "react-beautiful-dnd": "^13.1.1",
+    "uuid": "^9.0.0",
+    "yup": "^0.32.11"
+  },
+  "devDependencies": {
+    "@axinom/mosaic-portal": "0.31.0-rc.8",
+    "@dbeining/react-atom": "^4.1.21",
+    "@graphql-codegen/cli": "^2.13.4",
+    "@graphql-codegen/typescript": "^2.7.3",
+    "@graphql-codegen/typescript-operations": "^2.5.3",
+    "@graphql-codegen/typescript-react-apollo": "^3.3.7",
+    "@libre/atom": "^1.3.3",
+    "@types/node": "^18.11.19",
+    "@types/react": "^17.0.50",
+    "@types/react-beautiful-dnd": "^13.1.4",
+    "@types/react-dom": "^17.0.17",
+    "@types/react-router": "^5.1.19",
+    "@types/react-router-dom": "^5.3.3",
+    "autoprefixer": "^9.8.8",
+    "dotenv-webpack": "^7.1.1",
+    "env-cmd": "^10.1.0",
+    "fork-ts-checker-webpack-plugin": "^6.5.2",
+    "history": "^4.10.1",
+    "piral-cli": "^0.15.8",
```

Comment:
Ideally, I would compare the workflows versions between media workflows and channel workflows and align them.
Minimally, I would align the piral-related package versions, to eliminate a warning during build:all script run:
![image](https://github.com/Axinom/mosaic-media-template/assets/117624390/6a13df23-5d56-4a15-b003-87b0ba236035)